CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 # Mattermost Push Proxy Changelog
 
 
+## 5.9.0 Release
```

Comment:
This changelog hasn't been maintained since 5 years (last date was in 2019). And the latest release is 6.3, not 5.8.1. Safe to say, we can avoid writing this in the changelog, but rather mention everything in the PR message. After that, all the details should be captured in the GitHub releases page: https://github.com/mattermost/mattermost-push-proxy/releases